src/components/DatePicker.ts

```diff
@@ -53,16 +52,9 @@ export class DatePicker extends Component<DatePickerProps, DatePickerState> {
         );
     }
 
-    componentWillReceiveProps(newProps: DatePickerProps) {
-        this.setState({
-            printDate: `${format(newProps.dateAttribute, this.props.formatDate)}`
-        });
-    }
-
     private createDateInput() {
         return createElement(DateInput, {
-            dateAttribute: this.props.dateAttribute,
-            onChange: this.handleChange,
+            dateattribute: this.props.dateattribute,
```

Comment:
Use pascalCase for `dateattribute: this.props.dateattribute,`

Reply:
The pascal case gives warning during tests, Am going to rename it.

Reply:
Done